src/Utils/ArrayDiff.php

```diff
@@ -48,4 +50,39 @@ public function diff($oldData, $newData)
 
         return $diff;
     }
+
+    /**
+     * Compare the type and the property values of two objects.
+     * Return true if they are the same, false otherwise.
+     * If the type is the same and all properties are the same, this will return true, even if they are not the same instance.
```

Comment:
```suggestion
     * If the type is the same and all properties are the same, this will return true, even if they are not the same instance.
     *
     * @see https://www.php.net/manual/en/language.oop5.object-comparison.php.
```